src-browserify/core.js

```diff
@@ -6,46 +26,57 @@ cdb.CARTOCSS_VERSIONS = {
   '2.1.0': ''
 };
 cdb.CARTOCSS_DEFAULT_VERSION = '2.1.1';
+cdb.config = {};
 
-var Loader = require('./core/loader');
+var Loader = setupLoader(cdb);
 
-cdb.config = {};
-cdb.core = {
-  Profiler: require('./core/profiler'),
-  util: require('./core/util'),
-  Loader: Loader
-};
-cdb.Image = require('./vis/image.js');
-cdb.SQL = require('./api/sql');
-cdb.Tiles = require('./api/tiles');
+var BackboneEvents = window.Backbone ? window.Backbone.Events : BackboneIsch.Events
+var _Promise = setup_Promise(BackboneEvents);
 
-cdb.vis = {
-  Loader: Loader
-};
+cdb.core = {};
+cdb.core.Profiler = Profiler;
+cdb.core.util = util;
+cdb.core.Loader = Loader;
 
-cdb._Promise = require('_Promise');
+var globalJQuery = window.$ || window.jQuery;
 
-if (typeof window !== 'undefined') {
-  window.cartodb = cdb;
+var SubLayerBase = setupSubLayerBase(BackboneEvents);
+var HttpSubLayer = setupHttpSubLayer(SubLayerBase);
+var BackboneModel = window.Backbone ? window.Backbone.Model : null;
+var CartoDBSubLayer = setupCartoDBSubLayer(SubLayerBase, BackboneModel);
+var SubLayerFactory = setupSubLayerFactory(CartoDBSubLayer, HttpSubLayer);
+var MapBase = setupMapBase(SubLayerFactory, {
+  jQueryAjax: globalJQuery ? globalJQuery.ajax : undefined,
+  reqwestCompat: reqwest.compat
+});
+var LayerDefinition = setupLayerDefinition(MapBase, cdb.CARTOCSS_DEFAULT_VERSION);
```

Comment:
Yes, you're right on that, but to avoid having to _know_ that the version needs to be set in various places I think it's clearer to set the version in one place and pass it explicitly like this.